solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -102,6 +111,19 @@ public void call(ClusterState state, ZkNodeProps message, NamedList<Object> resu
     split(state, message, results);
   }
 
+  /**
+   * Shard splits start here and make additional requests to the host of the parent shard.
+   *
+   * <p>The sequence of requests is as follows: 1. Verify that there is enough disk space to create
```

Comment:
This isn't formatted as originally intended.

Reply:
yes, after running `./gradlew tidy`

Reply:
understood but it needs to be altered (e.g. by adding HTML elements) to represent the _intended_ display.